packages/xo-server-netbox/src/slugify.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Transforms a string into a slug with pattern ^[-a-zA-Z0-9_]+$
```

Comment:
```suggestion
 * Transforms a string into a slug with pattern ^[-a-zA-Z0-9]+$
```

Reply:
This is the pattern required by Netbox. My transformation does fit in that pattern even though I'm never using `_`. We might want to change the transformation in the future so I'd rather keep Netbox's requirement in the function's description to make sure we still respect that even if we decide to use `_`.

Reply:
I updated the description to make that clear.